entitlement-api/src/main/java/com/hevelian/identity/entitlement/api/PAPController.java

```diff
@@ -59,13 +58,27 @@ public PAPPolicy getPolicy(@Valid @RequestBody PolicyIdDTO papPolicyIdDTO)
     return papService.getPolicy(papPolicyIdDTO.getPolicyId());
   }
 
+  @RequestMapping(path = "/tryPolicy", method = RequestMethod.POST)
+  public PrimitiveResult<String> tryPolicy(@Valid @RequestBody TryPAPPolicyRequestDTO tryPAPPolicy)
```

Comment:
tryPolicy(@RequestBody EntitlementRequestDTO, @RequestParam policyId)

Reply:
Don't forget to delete DTO class

Reply:
Fixed